.github/workflows/build.yml

```diff
@@ -1,6 +1,16 @@
 name: Continuous Integration Checks
 
-on: [push, pull_request]
+on:
+  push:
+    branches-ignore:
+      - master
+  pull_request:
+    branches-ignore:
+      - master
```

Comment:
Do we mean `main` here? Also does this mean merges to main won't kick off CI anymore?